en/user_guide/keyword_management.rst

```diff
@@ -0,0 +1,13 @@
+Keyword management
+==================
+
+Open Orchestra allows to attach keywords to content or media. They describe the content and allow to manage it.
+In the keyword panel, you can create or destroy a keyword. If you destroy it, the keyword will stay in content type that already using it.
```

Comment:
``If you use a NoSql database and you destroy a keyword, it will still stay in the content where it is used``